pkg/app/app.go

```diff
@@ -133,6 +146,23 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	}
 }
 
+// ValidateUpDown validates flags for Up and Down
+func ValidateUpDown(cmd *cobra.Command, opt *kobject.ConvertOptions) {
```

Comment:
I think ValidateUpDown could be worded differently.

How about ValidateComposeFile?